transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -34,6 +34,7 @@ transactions.enable-statistics=Whether transaction statistics should be gathered
 transactions.enable-statistics.deprecated=Use statistics-enabled.
 transactions.enable-tsm-status=Whether the transaction status manager (TSM) service, needed for out of process recovery, should be provided or not.
 transactions.default-timeout=The default timeout for a transaction managed by the transaction manager.
+transactions.maximum-timeout=The maximum timeout for a transaction managed by the transaction manager.
```

Comment:
This may need to be a bit more verbose. It looks like this value is only used _if_ the `default-timeout` is set to 0.

Reply:
+1

Reply:
OK, I will change this description to make it clear.